policykit/templates/policyadmin/init_starterkit_slack.html

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
it'd be preferable to have one template for all integrations. this has a lot of code duplication